Initial commit to create the barebones stats component and stats being

ldalton02 opened the pull request:
passed down from landing page.

Adds some css into stats.css to style the bar.

Uses Math.random to generate random numbers from 1 to 10 for initial
stats.

ldalton02 commented:
closes #22

ldalton02 commented:
Let me know what you think, I can always add more changes/change font if it doesn't look close enough to figma

nendow02 reviewed (approved):
lgtm, I’m going to request @reginawang3495 to quickly look over this and @archishadatta too just because there’s some css in here. Nice job Luke!

archishadatta commented on src/styles/stats.scss:
```diff
@@ -1,12 +1,46 @@
+}
+
+.stats-display:nth-child(2) { 
+  color: #ffab00;
```
Use scss variables for the colors

archishadatta commented on src/styles/stats.scss:
```diff
@@ -1,12 +1,46 @@
   bottom: 50px;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
   height: 50px;
```
```suggestion
  height: 50px;
  padding: 1em;
```

reginawang3495 reviewed (changes requested):
oops!! I forgot to submit the review :)

ldalton02 commented:
My bad on late response time. I changed Stats.tsx to create an object out of the props and then map the object to display all of the stats. Also moved the random number generating function to common/math.js in the src folder

reginawang3495 reviewed (changes requested):
I dmed some things to consider changing as well!!

reginawang3495 reviewed (commented):
Quick glance — shape of PR looks really great to me!!

reginawang3495 commented on src/components/shared/Stats.tsx:
```diff
@@ -1,13 +1,29 @@
 import '../../styles/stats.scss';
+
+type statsObject = {
```
I would make a playerStats object somewhere and use it both here and in the state of App.tsx

reginawang3495 commented on src/components/shared/Stats.tsx:
```diff
@@ -1,13 +1,29 @@
+export default function Stats(props: statsProps): JSX.Element {
+  const { playerStats } = props;
+
+  const renderStats = () => {
```
Since there isn’t much code and this is only used once, I’d prefer to in-line this

reginawang3495 reviewed (changes requested):
A couple small ideas to improve + would recommend running “yarn lint-fix”

reginawang3495 reviewed (approved):
looks great!! a few tiny changes, then feel free to merge :)

some small formatting issues (like extra lines, and not having a new line at the end of a file), but not a big deal. Feel free to ask why as always!

ldalton02 commented on src/common/types.ts:
```diff
@@ -0,0 +1,3 @@
+export type statsObject = {
+  [key: string]: number,
+}
```
It might not always be the same in the future but good to manually define it to prevent issues
